src/Utility/ComposerUtils.php

```diff
@@ -89,13 +89,19 @@ public function __construct(Composer $composer, IOInterface $io)
     private function getPreferredInstallChanged(): array
     {
         $config = $this->configFile->read();
+        if (!is_array($config)) {
+            return [];
+        }
 
-        if (
-            !is_array($config)
-            || !is_array($config[self::EXTRA] ?? null)
-            || !is_array($config[self::EXTRA][self::EXTRA_BASE] ?? null)
-            || !is_array($config[self::EXTRA][self::EXTRA_BASE][self::EXTRA_PREFERRED_INSTALL_CHANGED] ?? null)
-        ) {
+        if (!is_array($config[self::EXTRA] ?? null)) {
+            return [];
+        }
+
+        if (!is_array($config[self::EXTRA][self::EXTRA_BASE] ?? null)) {
+            return [];
+        }
+
+        if (!is_array($config[self::EXTRA][self::EXTRA_BASE][self::EXTRA_PREFERRED_INSTALL_CHANGED] ?? null)) {
```

Comment:
This last one already covers the previous ones, courtesy of ??.  Just the last one is sufficient.  cf: https://3v4l.org/ZTbjY